### ISSUE TITLE
Unit Tests Always Fail With "oneOfType" Warning

### ISSUE THREAD

**Issue by JimTheMan** [adazzle/react-data-grid#914]
- Have you already searched for similar issues? Please help us out and double-check first!

This problem is also mentioned in this issue: https://github.com/adazzle/react-data-grid/issues/858

- Also, please don't be that person who deletes this template. It's here for a reason.

- Thanks!

---

### Which version of React JS are you using?

✅ Officially supported ✅
- [ ] v15.4.x

⚠️ Not officially supported, expect warnings ⚠️
- [x] v15.5.x

☣️ Not officially supported, expect warnings and errors ☣️
- [ ] v16.x.x

---

### Which browser are you using?

✅ Officially supported ✅
- [ ] IE 9 / IE 10 / IE 11
- [ ] Edge
- [x] Chrome

⚠️ Not officially supported, but "should work" ⚠️
- [ ] Firefox
- [ ] Safari

---

### I'm submitting a ...

- [x] 🐛 Bug Report
- [ ] 💡 Feature Request

> 👋 Need general support? Not sure about how to use React itself, or how to get started with the Grid?
> Please do not submit support request here. Instead see

> https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md

---

### Issue Details

Please include:
- What the current behavior is

When running unit tests I always get test failures and this red error message: 
```
  console.error node_modules\fbjs\lib\warning.js:35
    Warning: Invalid argument supplid to oneOfType. Expected an array of check functions, but received undefined at index 0.
```

- What the desired behavior is

no errors

- (If Bug) Steps to reproduce the issue

Include react-data-grid into any create-react-app application and run tests (npm test).

- (If Feature) The motivation / use case for the feature

We especially love screenshots / videos of problems, and remember
The Best Issue Is A Pull Request™

**Comment by amanmahajan7** [comment]
This has been fixed in malonecj/react-autocomplete#9

**Comment by wKovacs64** [comment]
If anyone finds themselves here and still getting the warnings, see [my comment](https://github.com/adazzle/react-data-grid/issues/858#issuecomment-332910720) in #858.
